src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -1269,13 +1269,15 @@ public bool TryGetConstantValue(TexlNode node, out string nodeValue)
                 case NodeKind.FirstName:
                     // Possibly a non-qualified enum value
                     var firstNameNode = node.AsFirstName();
-                    var firstNameInfo = GetInfo(firstNameNode);
-                    if (firstNameInfo.Kind == BindKind.Enum)
+                    if (context.NameResolver.Lookup(firstNameNode.Ident.Name, out var firstNameInfo, NameLookupPreferences.None))
```

Comment:
>context.NameResolver [](http://example.com/codeflow?start=24&length=20)

This is an instance method on Binder? 
But also takes a check context... do we need that if we're an instance method?
Can we just get NameResolver from the binder?

Reply:
This is a partial transition to being static. The signature in PA client is also updated. However, the second code block below (for DottedNamedNode) still does not work. This partial change confirms which code block was causing the problem.